gradle/libraries.toml

```diff
@@ -78,7 +78,7 @@ gr8 = { group = "com.gradleup", name = "gr8-plugin", version = "0.7" }
 # Keep in sync with MIN_GRADLE_VERSION
 gradle-api-min = { group = "dev.gradleplugins", name = "gradle-api", version = "5.6" }
 gradle-japicmp-plugin = { group = "me.champeau.gradle", name = "japicmp-gradle-plugin", version = "0.2.8" }
-gradle-publish-plugin = { group = "com.gradle.publish", name = "plugin-publish-plugin", version = "0.12.0" }
+gradle-publish-plugin = { group = "com.gradle.publish", name = "plugin-publish-plugin", version = "1.1.0" }
```

Comment:
That was required because the older plugin was calling a `setClassifier` method that wasn't working anymore